src/main/java/frc/robot/robotcontainers/NewRobotContainer.java

```diff
@@ -62,11 +65,13 @@ public NewRobotContainer() {
   private void configureButtonBindings() { 
     
     
-    /* Driver Controls */
-    
+    /* Driver Controls */  
     driver.start().onTrue(SwerveSubsystem.getInstance().setHeadingCommand(new Rotation2d(0)));
   
     driver.back().whileTrue(SwerveSubsystem.getInstance().setLockWheelsInXCommand());
+    driver.b().onTrue(SwerveSubsystem.getInstance().setOdometryCommand(new Pose2d(3,3,new Rotation2d(0))));
```

Comment:
Add a note/comment before line 72, to indicate the following b and a buttons are for vision test now.